lib/stat_tracker.rb

```diff
@@ -1,6 +1,54 @@
-class StatTracker
+require 'csv'
+require_relative 'game'
+require_relative 'team'
 
-  def self.from_csv(location)
+class StatTracker 
+  #include modules
 
+  def self.from_csv(locations)
+    StatTracker.new(locations)
   end
-end
\ No newline at end of file
+
+  attr_reader :game_data, :team_data, :game_teams_data
+
+  def initialize(locations)
+    @game_data = CSV.read locations[:games], headers: true, header_converters: :symbol
+    @team_data = CSV.read locations[:teams], headers: true, header_converters: :symbol
+    @game_teams_data = CSV.read locations[:game_teams], headers: true, header_converters: :symbol
```

Comment:
Reading CSVs in initialize is difficult to test.   We would have to have three readable CSVs each time we instantiate a stat_tracker.  Could we bring these 3 lines (15-17) up to class method (.from_CSV)

Additionally we can pass into `.new` list of Game, Team, GameTeam rather then the hashes that come out of the CSV.

Reply:
Just noting that together, we could not figure out how to make this work. A collective decision was make to keep these in the initialize!